x/ccv/provider/keeper/power_shaping_test.go

```diff
@@ -1071,101 +1081,3 @@ func TestUpdatePrioritylist(t *testing.T) {
 	}
 	require.Equal(t, expectedPrioritylist, providerKeeper.GetPriorityList(ctx, consumerId))
 }
-
-func TestFilterAndSortPriorityList(t *testing.T) {
```

Comment:
Why remove this? You can mock the data for the consumer ID and still have this test with minor adjustments, right?

Reply:
Yes, I agree, but it doesn’t seem to add much value compared to TestCapValidatorSet since FilterAndSortPriorityList is also tested through it. I can easily return it if needed.